src/codemagic/apple/app_store_connect/apps/apps.py

```diff
@@ -77,19 +77,19 @@ def list_builds(self, app: Union[LinkedResourceData, ResourceId]) -> List[Build]
         Warning! As of 11.08.21 pagination does not work as expected for this API endpoint. See
         https://github.com/codemagic-ci-cd/cli-tools/pull/140 for more information.
         """
-        if isinstance(app, App):
+        if isinstance(app, App) and app.relationships is not None:
             url = app.relationships.builds.links.related
-        else:
+        if url is None:
```

Comment:
Here and elsewhere `url` can be undefined which will cause a `NameError`.

Reply:
oops forgot the `url = None` in that file